bson/__init__.py

```diff
@@ -244,7 +244,7 @@ def _raise_unknown_type(element_type: int, element_name: str) -> NoReturn:
 
 
 def _get_int(
-    data: Any, view: Any, position: int, dummy0: Any, dummy1: Any, dummy2: Any
+    data: Any, _view: Any, position: int, dummy0: Any, dummy1: Any, dummy2: Any
```

Comment:
Why did ruff rename "view" -> "_view"? It seems out of place.

Reply:
I did that manually, `dummy` is explicitly part of `dummy-variable-rgx`.